build.gradle.kts

```diff
@@ -66,6 +67,9 @@ tasks.withType<Test> {
 
 allprojects {
     repositories {
+        // temporary repo containing mockative 3.0.1 with a fix for a bug https://github.com/mockative/mockative/issues/143 is uploaded
+        // until mockative releases a new version with a proper fix
+        maven(url = "https://raw.githubusercontent.com/saleniuk/mockative/fix/duplicates-while-merging-dex-archives-mvn/release")
```

Comment:
afaik this will cause issues when releasing to fdroid, lets be careful and wait for the fix to be merged first

Reply:
Can you tell a bit more how would it cause problems? It's just another maven repo url so it should work just like any other 🤔 
Other options are: wait with updating to kotlin 2.1 until mockative releases version with that fix or disabling big part of our tests (ios, js, most of android instrumented)

Reply:
I think I found what you mean - is it because f-droid has this policy of trusting only some "well known" repositories so using some custom one may be restricted?
So back to my question - should I just revert and disable all these tests for the time being?